lib/loader/serialize/serial_type.cpp

```diff
@@ -86,6 +116,14 @@ Serializer::serializeValType(const ValType &Type, ASTNodeAttr From,
   case TypeCode::Ref:
   case TypeCode::RefNull:
     return serializeRefType(Type, From, OutVec);
+  case TypeCode::NullExternRef:

```

Comment:
As commented before, the following cases will always not be reached because the `getCode()` for `valtype` will always be the cases above.
REF: https://github.com/WasmEdge/WasmEdge/blob/master/include/common/types.h#L63-L85